juniper_official/Interfaces/interface-flaps-iagent.rule

```diff
@@ -0,0 +1,126 @@
+/*
+ * Detects interface flaps and notifies when anomalies are found.
+ */
+healthbot {
+    topic interface.statistics {
+        description "Monitors and notify interface statistics i.e. link state, flaps, neighbor state, input & output traffic and errors";
+        synopsis "Interface statistics analyzer";
+        rule check-interface-flaps-iagent {
+            keys name;
+            synopsis "Link flaps detector";
+            description "Collects link flap count periodically and notifies when flap count increases";
+            sensor interface-stats {
+                synopsis "Interfaces netconf based iAgent sensor definition";
+                description "iAgent sensor defined in yml file using netconf rpc/cli command and defined fields using pyez tables&views to collect data from network device";
+                iAgent {
+                    file interface-details.yml;
+                    table intfaceStats;
+                    frequency 30s;
+                    args interface_name {
+                        arg-value "{{interface-name}}";
+                    }
+                }
+            }
+            field flaps-threshold {
+                constant {
+                    value "{{flaps-threshold-variable}}";
+                }
+                type integer;
+                description "Flap count increase threshold";
+            }
+            field name {
+                sensor interface-stats {
+                    path name;
+                }
+                type string;
+                description "Interfaces to be monitored";
+            }
+            trigger link-flap {
+                synopsis "Interface flap  kpi";
+                description "Sets health based on the change in flap count";
```

Comment:
Add the trigger frequency atleast 1.5 times of sensor frequency so we can avoid dashboard flickerring.

Reply:
modified trigger frequency as 90s